File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/spot/SpotCancelOrderReq.java
package com.bitget.openapi.dto.request.spot;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.io.Serializable;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: cancel the order request
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SpotCancelOrderReq implements Serializable {

    private static final long serialVersionUID = -1L;

    /**
     * Currency pair
     */
    private String symbol;

    /**
     * Order Id
     */
    private String orderId;
}

File: bitget-node-sdk-api/src/lib/model/spot/order/SpotCancelOrderReq.ts
export class SpotCancelOrderReq {
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * Order Id
     */
    private _orderId!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderId(): string {
        return this._orderId;
    }

    set orderId(value: string) {
        this._orderId = value;
    }
}